go.mod

```diff
@@ -120,7 +119,34 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-require (
-	github.com/spf13/afero v1.6.0 // indirect
-	sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20230208013708-22718275bffe // indirect
+require sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20230208013708-22718275bffe
+
+require github.com/spf13/afero v1.6.0 // indirect
+
+replace (
```

Comment:
I didn't really question this last time - why is this here?

Reply:
(The `replace` clause, just to clarify)

Reply:
Hi, @adrianludwin 
In conclusion, the replace clause seems to be unnecessary.
It was added #283 as a workaround and has to be unneeded in #286.
Fundamentally this was my mistake... sorry.

So I removed it in #294 and please check it.

Reply:
Thanks for fixing it, can you add #294 to this PR as well? Thanks!

Reply:
I did it, please check. Thanks!

Reply:
@keisukesakasai Thanks your add:)